vendor/assets/javascripts/jquery.j-toker.js

```diff
@@ -655,6 +655,7 @@
         delete opts.config;
 
         opts.confirm_success_url = config.confirmationSuccessUrl();
+        opts.firsttime = true
```

Comment:
e nasol daca vrem sa facem update ca nu o sa retinem sa modificam asta. @ungureanu-bogdan-sorin ti-am zis daca inca nu exista o modalitate sa facem ce vrem putem sa mai asteptam un pic pana fac ei implementarea in devise_token_auth sau sa ne apucam sa facem Pull requesturi si sa bage ei schimbarile de la noi. Dar e ok si sa renuntam pentru un moment la schimbarea asta